### PR TITLE
Load Configuration file only once, instead of per tweet

### PR DESCRIPTION
**Pull request by jlpedrosa**
Configuration file is static and does not change during the execution f the program, reloading it per tweet has a mayor impact on performance
added .gitignore (the file patterns added to this file are not commited).
formatted .json configuration file for better readability.
Parameterized file name for tickers.

### CODE REVIEW

**Comment by jlpedrosa** [comment]
Hi @AndreaRizzi314  I did not test the code, it is more about guidelines. It should be close to working.
If you have any troubles happy to help.

There are a bunch of other optimizations/improvements that we can tackle later.

**Comment by AndreaRizzi314** [comment]
I'm looking through the code now, thanks so much for the help!

File: TwitterAnalysingBot/Twitter_API.py [hunk 1]
```diff
@@ -12,29 +12,32 @@
...
+    #Create a python dictionary with the reset hypescores of all the stocks
+    Tickers = dict.fromkeys([x.strip() for x in lines], 0)
 
-#Create a python dictionary with the reset hypescores of all the stocks
-Tickers = {}
-for Ticker in lines:
-    Tickers[Ticker] = 0
+class StdOutListener(StreamListener):
 
+    def __init__(self, configurationFilePath):
```
Comment (AndreaRizzi314) [inline TwitterAnalysingBot/Twitter_API.py:28]: Im not familiar with the __init__, what does it do?

File: TwitterAnalysingBot/Twitter_API.py [hunk 2]
```diff
@@ -12,29 +12,32 @@
...
+        #Load configuration file in memory once with the constructor.
+        with open('Configuration.json', 'r') as f:
```
Comment (AndreaRizzi314) [inline TwitterAnalysingBot/Twitter_API.py:30]: Could I replace the "Configuration.json" file path with the one I input in the constructor?

**Review by AndreaRizzi314 — APPROVED** [review]

**Pull request merged**